main/guides/coreeval/local-testnet.md

```diff
@@ -0,0 +1,33 @@
+# Create a Local Testnet
+
+To create, start, and deploy an Agoric contract to a local Agoric Testnet, do the following:
+
+
+1. Install *Go* if needed.
+
+	```
+    brew install go
+    ```
+2. Navigate to `<agoric-sdk>/packages/cosmic-swingset` and run `make`.
+3. Prepare the chain and solo nodes.
+
+	```jsx
+	make scenario2-setup BASE_PORT=8000 NUM_SOLOS=0
+	```
+4. Start the chain.
+
+	```jsx
+	make scenario2-run-chain
+	```
+5. Switch to another terminal and wait for the first block to be produced.
+6. Navigate to `<agoric-sdk>/bin` and start the client.
```

Comment:
```suggestion
6. Navigate to `<agoric-sdk>/bin` and submit the governance proposal.
```

It doesn't start a client running.

Reply:
And this isn't part of starting a local chain; it shouldn't be under than heading.

Reply:
I made the changes suggested above.

In addition, to address your second issue, (i.e., "it shouldn't be under that heading") I change the title of the topic to "Deploy a Governance Proposal to a Local  Testnet" so that it properly reflects all the actions described in the topic.

Reply:
> I made the changes suggested above.

Did you push it yet? The most recent commit I see in this PR is on May 24 b22d2dd8d02b7b8b3bf6210d1d763eb4565687f8 .